aeron-samples/src/main/java/io/aeron/samples/AeronStat.java

```diff
@@ -195,6 +211,24 @@ private static void workLoop(final long delayMs, final Runnable outputPrinter)
         while (running.get());
     }
 
+    private static void telegrafOutput(final CncFileReader cncFileReader, final StringJoiner telegrafMetric)
+    {
+        final CountersReader counters = cncFileReader.countersReader();
+        counters.forEach(
+            (counterId, typeId, keyBuffer, label) ->
+            {
+                if (counterId <= 24)
+                {
+                    final long cv = counters.getCounterValue(counterId);
+                    final String cl = counters.getCounterLabel(counterId).replaceAll("( |,|\\.|-)", "_").toLowerCase();
```

Comment:
It would be better to use pre-compiled Pattern and Matcher.

Reply:
Done.